Assets/_BForBoss/Scripts/Managers/WorldManager.cs

```diff
@@ -74,6 +93,21 @@ private void SetupLeaderboardViews()
             _perigonAnalytics.StartSession(SystemInfo.deviceUniqueIdentifier);
         }
 
+        private void Update()
+        {
+            if (Keyboard.current[PauseKey].wasPressedThisFrame)
```

Comment:
Right now the InputSystem has the Escape key attached to the FirstPersonCharacter, where pressing it auto unlocks the cursor! So kinda makes unpausing a bit wonky here cause the cursor is supposed to lock but sometimes doesn't

If its cool @m4rtint can I remove the InputSystem for Escape?

Reply:
not yet, it might break some other stuff, i'll handle it when i connect stuff.